src/com/google/javascript/jscomp/ProductionCoverageInstrumentationCallback.java

```diff
@@ -60,14 +64,42 @@
   private final ParameterMapping parameterMapping;
   boolean visitedInstrumentCodeFile = false;
 
-  /** Stores the name of the current function that encapsulates the node being instrumented */
-  private String cachedFunctionName = "Anonymous";
+  private enum Type {
+    FUNCTION("Type.FUNCTION"),
```

Comment:
You don't need value given that it's just repeats name. Instead you get enum name: `Type.FUNCTION.name()` which returns `FUNCTION`.

Reply:
In this particular case, the value I was passing is not a repeat of the enum name `"Type.FUNCTION"` vs `FUNCTION` but thinking about it more, it could be and probably should be since it doesn't make it any easier to convert back to an enum later so I changed it.